packaging/MacSDK/patches/fsharp-portable-pdb.patch

```diff
@@ -15,7 +15,12 @@ index 34d716485..ec99e87ed 100644
 +        <_ConfigurationNameTmp>$(ConfigurationName)</_ConfigurationNameTmp>
 +        <_ConfigurationNameTmp Condition="'$(ConfigurationName)' == ''">$(Configuration)</_ConfigurationNameTmp>
 +
++        <!-- _DebugFileExt is not an upstream msbuild feature yet -->
++        <_DebugFileExt>.pdb</_DebugFileExt>
++        <_DebugFileExt Condition="'$(FscDebugFileExt)' != ''">$(FscDebugFileExt)</_DebugFileExt>
++
 +        <DebugType Condition="'$(OS)' != 'Windows_NT' And ('$(DebugSymbols)'=='True' or ('$(DebugSymbols)'=='' And '$(_ConfigurationNameTmp)'=='Debug'))">portable</DebugType>
++        <DebugType Condition="'$(FscDebugType)' != ''">$(FscDebugType)</DebugType>
```

Comment:
it's not clear to me why we need FscDebugType, why not add a `'$(DebugType) == ''` check to the Condition above and let users set DebugType ?

Reply:
DebugType is set to `full` higher up in the chain

Reply:
Ok, makes sense then.